src/library/borrowitem/BorrowItemControl.java

```diff
@@ -0,0 +1,125 @@
+package library.borrowitem;
+import java.util.ArrayList;
+import java.util.List;
+
+import library.entities.Item;
+import library.entities.Library;
+import library.entities.Loan;
+import library.entities.Patron;
+
+public class BorrowItemControl {
+    
+    private BorrowItemUI ui;
+    
+    private Library library;
+    private Patron patron;
+    private enum CONTROL_STATE { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED }
+    private CONTROL_STATE state;
```

Comment:
@amrmosit I missed this, CSG 4 states this should be camel case.

Reply:
Ok. I will do this modification at the end of iteration 2 with any other required modifications. Updated Iteration2 plan